### ISSUE TITLE
Error in translating google.protobuf.Duration in mixer adapter config

### ISSUE THREAD

**Issue by venilnoronha** [istio/istio#8363]
**Describe the bug**
When deploying an out-of-process adapter, the following log is observed for a `google.protobuf.Duration` field when it's specified in yaml in the string format.
```
2018-08-30T04:04:19.837110Z	error	field 'flushInterval' is of type 'string' instead of expected type 'google.protobuf.Duration'
```

**Expected behavior**
As per the protobuf [documentation](https://github.com/protocolbuffers/protobuf/blob/d36c0c538a545fac5d9db6ba65c525246d4efa95/src/google/protobuf/duration.proto#L92-L100), it should be possible to translate the field from a string.

**Steps to reproduce the bug**
Follow the out-of-process [adapter guide](https://github.com/istio/istio/wiki/Mixer-Out-of-Process-Adapter-Walkthrough) and include a `google.protobuf.Duration` field in the configuration.

The proto definition for the field is as follows.
```proto
google.protobuf.Duration flush_interval = 2 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true];
```

In the failure scenario, the yaml configuration for the field was as follows.
```yaml
  params:
    flushInterval: 5s
```

It worked, however, when the field was expanded to match the `google.protobuf.Duration` type:
```yaml
  params:
    flushInterval:
      seconds: 5
```

**Version**
Istio:
```
Version: 1.0.1
GitRevision: 42773aacced474d97159902d20579a25b1f98106
User: root@832d5020b1d4
Hub: gcr.io/istio-release
GolangVersion: go1.10.1
BuildStatus: Clean
```

Kubernetes:
```
Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.0", GitCommit:"91e7b4fd31fcd3d5f436da26c980becec37ceefe", GitTreeState:"clean", BuildDate:"2018-06-27T22:30:22Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.0", GitCommit:"fc32d2f3698e36b93322a3465f63a14e9f0eaead", GitTreeState:"clean", BuildDate:"2018-03-26T16:44:10Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
```

**Is Istio Auth enabled or not?**
No

**Environment**
Minikube 0.28 (macOS)

**Note**
The same issue was observed when trying to write a test as described in the [guide](https://github.com/istio/istio/wiki/Mixer-Out-of-Process-Adapter-Walkthrough#step-6-write-test-and-validate-your-adapter).

**Comment by mandarjog** [comment]
This is a dup of https://github.com/istio/istio/issues/7645

It is fixed in master. Do you need the fix on 1.0.x stream? Otherwise it will be in 1.1.

**Comment by venilnoronha** [comment]
Yes, that'd be great. I'm relying on 1.0.x for developing an out-of-process adapter which I want to release shortly.
